src/core/reactor.cc

```diff
@@ -960,6 +960,9 @@ reactor::task_queue::register_stats() {
         }, sm::description("Total amount in milliseconds we were in violation of the task quota"),
            {group_label}),
     });
+
+    register_net_metrics_for_scheduling_group(new_metrics, _id, group_label);
+
```

Comment:
This is the only call, so new groups won't have metrics.

Reply:
Wait that's completely wrong.

Reply:
Sorry I don't follow. This gets called whenever a new group is added (or renamed). It works effectively the same way as scheduling groups get their metrics right now.

Reply:
"that's completely wrong" referred to my previous comment